packages/probers/cypress/e2e/signIn.cy.ts

```diff
@@ -30,6 +30,16 @@ describe('Sign In', () => {
     assertOnSignInPage()
   })
 
+  it.only('can navigate to sign-in after entering email in sign-up', () => {
```

Comment:
I came here to copy homework and I saw this. Was this `.only` intentional to leave in? 👀